### PR TITLE
Create yabellini.netlify.app.json

### PR DESCRIPTION
**Pull request by yabellini**
Adding my blog ! :-)

### CODE REVIEW

File: blogs/yabellini.netlify.app.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "language": "en", 
+  "authors": [ 
+    {
+      "name": "Yanina Bellini Saibene", 
+      "social_media": [{
+         "twitter": "yabellini",
+         "github": "yabellini",
+         "instagram": "yabellini",
+         "youtube": "channel/UCA7IUNba2IjQfrVVjh2Whmw",
+         "tiktok": "",
```
Comment (Athanasiamo) [inline blogs/yabellini.netlify.app.json:17]:
```suggestion
```

File: blogs/yabellini.netlify.app.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+         "periscope": "",
```
Comment (Athanasiamo) [inline blogs/yabellini.netlify.app.json:18]:
```suggestion
```

File: blogs/yabellini.netlify.app.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+         "researchgate": "username",
+         "website": "https://yabellini.netlify.app/",
+         "linkedin": "/in/yabellini/",
+         "facebook": "",
```
Comment (Athanasiamo) [inline blogs/yabellini.netlify.app.json:22]:
```suggestion
```

File: blogs/yabellini.netlify.app.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+         "orcid": "0000-0002-4522-7466",
+         "meetup": "end-url"
```
Comment (Athanasiamo) [inline blogs/yabellini.netlify.app.json:24]:
```suggestion
```

File: blogs/yabellini.netlify.app.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+         "github": "yabellini",
+         "instagram": "yabellini",
+         "youtube": "channel/UCA7IUNba2IjQfrVVjh2Whmw",
+         "tiktok": "",
+         "periscope": "",
+         "researchgate": "username",
+         "website": "https://yabellini.netlify.app/",
+         "linkedin": "/in/yabellini/",
+         "facebook": "",
+         "orcid": "0000-0002-4522-7466",
```
Comment (Athanasiamo) [inline blogs/yabellini.netlify.app.json:23]:
```suggestion
         "orcid": "0000-0002-4522-7466"
```

File: blogs/yabellini.netlify.app.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "name": "Yanina Bellini Saibene", 
+      "social_media": [{
+         "twitter": "yabellini",
+         "github": "yabellini",
+         "instagram": "yabellini",
+         "youtube": "channel/UCA7IUNba2IjQfrVVjh2Whmw",
+         "tiktok": "",
+         "periscope": "",
+         "researchgate": "username",
```
Comment (Athanasiamo) [inline blogs/yabellini.netlify.app.json:19]:
```suggestion
```

File: blogs/yabellini.netlify.app.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
+{
+  "title": "Yanina Bellini Saibene", 
+  "subtitle": "", //optional
```
Comment (Athanasiamo) [inline blogs/yabellini.netlify.app.json:3]:
```suggestion
```

**Review by Athanasiamo — COMMENTED** [review]
just removing some empty fields

**Comment by yabellini** [comment]
Thanks!

**Comment by Athanasiamo** [comment]
Thank you for adding your blog to the list and for making free and open R content 😊 🥳 🎉

**Pull request merged**